dmotion:
  ankle_y:
    data: [0, 1.6779068745192227, 1.9743127835601815, 2.29866795397102, 3.1690484936969177, 3.6067643380015957, 4, 5.5, -14, -12.379867781315268, -10.644993363738045, -8.7469247916779818, -12.187986682773651, -13.225932642486972, -13.800000000000001, -14.41790454592352, 0, 2.717920410478345, 3.8473163199525895, 1.4228560774318622, -1.2845565621759976, -2.3108636538823957, -1.505120997689533, 0]